kimchi/src/circuits/polynomials/range_check/circuitgates.rs

```diff
@@ -239,44 +237,44 @@ where
                 .collect::<Vec<E<F>>>(),
         );
 
-        // 2) Constrain that the combined sublimbs equals the limb l2 stored in w(0) where
-        //    l2 = lp0 lp1 lp2 lp3 lc0 lc1 lc2 lc3 lc4 lc5 lc6 lc7 lc8 lc9 lc10 lc11 lc12 lc13 lc14 lc15 lc16 lc17 lc18 lc19
+        // 2) Constrain that the combined limbs equals the value v2 stored in w(0) where
+        //    v2 = vp0 vp1 vp2 vp3 vc0 vc1 vc2 vc3 vc4 vc5 vc6 vc7 vc8 vc9 vc10 vc11 vc12 vc13 vc14 vc15 vc16 vc17 vc18 vc19
         //    in little-endian byte order.
         //
         //          Columns
         //          0    1   2   3   4   5    6    7    8    9    10   11   12   13   14
-        //    Curr  l2   lp0 lp1 lp2 lp3 lc0  lc1  lc2  lc3  lc4  lc5  lc6  lc7  lc8  lc9
-        //    Next                       lc10 lc11 lc12 lc13 lc14 lc15 lc16 lc17 lc18 lc19
+        //    Curr  v2   vp0 vp1 vp2 vp3 vc0  vc1  vc2  vc3  vc4  vc5  vc6  vc7  vc8  vc9
+        //    Next                       vc10 vc11 vc12 vc13 vc14 vc15 vc16 vc17 vc18 vc19
         //
-        // Check   l2 = lp0*2^0          + lp1*2^{12}       + ... + lp3*2^{36}       + lc0*2^{48}     + lc1*2^{50}     + ... + lc19*2^{66}
+        // Check   v2 = vp0*2^0          + vp1*2^{12}       + ... + vp3*2^{36}       + vc0*2^{48}     + vc1*2^{50}     + ... + vc19*2^{66}
         //       w(0) = w_curr(1)*2^0    + w_curr(2)*2^{12} + ... + w_curr(4)*2^{36} + w_curr(5)*2^48 + w_curr(6)*2^50 + ... + w_curr(14)*2^66
         //            + w_next(5)*2^{68} + w_next(6)*2^{70} + ... + w_next(14)*2^{86}
         // (1st part) = \sum i \in [1,5] 2^{12*(i - 1)}*w_curr(i) + \sum i \in [6,14] 2^{2*(i - 5) + 4*12}*w_curr(i)
         // (2nd part) + \sum i \in [5,14] 2^{2*(i - 5} + 68)*w_next(i)
 
         let mut power_of_2 = E::one();
-        let mut sum_of_sublimbs = E::zero();
+        let mut sum_of_limbs = E::zero();
 
-        // 1st part: Sum 12-bit sublimbs (row Curr)
+        // 1st part: Sum 12-bit limbs (row Curr)
         for i in 1..5 {
-            sum_of_sublimbs += power_of_2.clone() * witness_curr(i);
+            sum_of_limbs += power_of_2.clone() * witness_curr(i);
```

Comment:
similar question here as in `RangeCheck0`

Reply:
Same answer